# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/RTCRtpReceiver.cpp
# Commit: afe92bb8ee8e
# Full Hash: afe92bb8ee8e54127cbe8ae06356974d7841ff90
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Small fixes to RTCRtpReceiver and TransceiverImpl. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130068
# ==============================================================================

diff -r 1a244e507326 -r afe92bb8ee8e dom/media/webrtc/jsapi/RTCRtpReceiver.cpp
--- a/dom/media/webrtc/jsapi/RTCRtpReceiver.cpp	Wed Nov 04 07:25:46 2020 -0500
+++ b/dom/media/webrtc/jsapi/RTCRtpReceiver.cpp	Wed Nov 04 11:10:31 2020 -0500
@@ -428,7 +428,7 @@
   if (mJsepTransceiver->HasBundleLevel() &&
       (!mJsepTransceiver->mRecvTrack.GetNegotiatedDetails() ||
        !mJsepTransceiver->mRecvTrack.GetNegotiatedDetails()->GetExt(
-           webrtc::RtpExtension::kMIdUri))) {
+           webrtc::RtpExtension::kMidUri))) {
     mStsThread->Dispatch(
         NewRunnableMethod("VideoSessionConduit::DisableSsrcChanges", conduit,
                           &VideoSessionConduit::DisableSsrcChanges));